packages/components/src/tabs/tabs.tsx

```diff
@@ -16,6 +17,7 @@ import type { Ref } from 'react'
 type Variants = GetVariants<typeof TriggerBase>
 
 type Props = {
+  // type: TriggerProps['type']
```

Comment:
Didn't figure out right away how to make this type safe in way that `type` wouldn't have to be set on `TriggerProps` by consumers.